src/daft-connect/Cargo.toml

```diff
@@ -12,7 +12,6 @@ daft-scan = {workspace = true}
 daft-schema = {workspace = true}
 daft-table = {workspace = true}
 dashmap = "6.1.0"
-derive_more = {workspace = true}
```

Comment:
Is there a reason we don’t want to use `derive_more`?

Reply:
we were only using it for `#[derive(Constructor)]` but we needed a custom impl now.

Reply:
we can add it back when we need it again.